system/Session/Session.php

```diff
@@ -232,7 +291,7 @@ protected function configure()
 	 */
 	protected function initVars()
 	{
-		if ( ! empty($_SESSION['__ci_vars']))
+		if (!empty($_SESSION['__ci_vars']))
```

Comment:
Agreed. I haven't found the proper auto-format setting for this, and will have to be vigilant.